### ISSUE TITLE
Create a separate set of files in SVG format for laser cutting

### ISSUE THREAD

**Issue by Molkavi** [Wollivan/SquadronLeader#75]
I plan on printing everything on clear acrylic with a laser cutting machine. It's just "cheaper", faster, and more accurate than 3D printing. The laser-cutting process can also create enough bas relief to accommodate all token functionality.

Transparency on some of the tokens can also enable some pretty cool (untested) UI improvements and mechanics (e.g. stackable altitude tokens, customizable plane look/loadout, etc.)

Can we get the core fileset in SVG format?

**Comment by Wollivan** [comment]
This is a great idea! I will have a look at exporting this now!

**Comment by Wollivan** [comment]
Having some issues with exporting as SVGs from OpenSCAD.

Its moaning about the objects not being 2D, so if anyone knows a simple way to just flatten them and export I will have a go!


I also want to ask @Molkavi , do they need to be individual SVGs, or can you work from one that is the entire set all in one go? Either is fine!

**Comment by Molkavi** [comment]
I should have no problem with a single file, I can always break it down into individual components later.

I'll give 2D'ing it a shot in Blender, I've done that kinda thing before.

Hopefully it'll take.

**Comment by Molkavi** [comment]
So I just finished a quick and dirty SVG file set, only the bare necessities, and I can publish it if I get write access.

I think, however, that it'd be much less of a hassle going forward to just have the core file set as a 2D vector design and extrude those 2D designs into 3D pieces as necessary.

After all, most pieces are 2D designs with varying levels of extrusion, right?

**Comment by Wollivan** [comment]
Nice one! You should just be able to push to a branch and create a Pull request :)

Also good thought about the models. Once we have a definite set of unchanging models I will make this how it works!

**Comment by Molkavi** [comment]
I tried, says I don't have write access to the repo =(

On another note, if I can get some measurements for the pieces I feel pretty confident I can do the SVG templates in Illustrator.

**Comment by Wollivan** [comment]
Sorry I never did this @Molkavi, I will get this sorted in the next week!

**Comment by Molkavi** [comment]
No problem mate, just let me know so I can contribute, all the best!

**Comment by Wollivan** [comment]
@Molkavi I'm actively working on this again, so if you can DM me an address, I can send you. A copy, and you can measure the pieces, and have a reference. For the measurements at the moment, I'm working on changing the way the OpenSCAD files are organised, so there is/will be (depending on when you read this) a file called variables which will have all the measurements in mm in.

**Comment by Wollivan** [comment]
@Molkavi I'm actively working on this again, so if you can DM me an address, I can send you. A copy, and you can measure the pieces, and have a reference. For the measurements at the moment, I'm working on changing the way the OpenSCAD files are organised, so there is/will be (depending on when you read this) a file called variables which will have all the measurements in mm in.

**Comment by Molkavi** [comment]
Yeah, I was thinking of redoing all the templates on Blender using mm scale, in 2D, and then just have a separate file for 3D printers which is basically the original file with a couple mm of extrusion.

**Comment by Wollivan** [comment]
Hi Molkavi, I realised why it might not be working!

Did you fork the repo first? 

If you fork it, make the changes on the fork, then when you're done go to pull requests (again on your own repo) 

Hope that helps!
